packages/rocketchat-theme/assets/stylesheets/base.less

```diff
@@ -4294,6 +4294,14 @@ body:not(.is-cordova) {
 			font-weight: bold;
 		}
 	}
+	.video-flip {
+		-moz-transform: scaleX(-1);
+		-o-transform: scaleX(-1);
+		-webkit-transform: scaleX(-1);
+		transform: scaleX(-1);
+		filter: FlipH;
+		-ms-filter: "FlipH";
```

Comment:
@mhlavac we are using https://github.com/less/less-plugin-autoprefix so you don't need to prefix theses items